core/domain/topic_services.py

```diff
@@ -479,19 +471,19 @@ def _save_topic(committer_id, topic, commit_message, change_list):
     topic.validate()
 
     topic_model = topic_models.TopicModel.get(topic.id, strict=False)
-    if topic_model is None:
-        topic_model = topic_models.TopicModel(id=topic.id)
-    else:
-        if topic.version > topic_model.version:
-            raise Exception(
-                'Unexpected error: trying to update version %s of topic '
-                'from version %s. Please reload the page and try again.'
-                % (topic_model.version, topic.version))
-        elif topic.version < topic_model.version:
-            raise Exception(
-                'Trying to update version %s of topic from version %s, '
-                'which is too old. Please reload the page and try again.'
-                % (topic_model.version, topic.version))
+
+    # Topic model cannot be None as topic is passed as parameter here and that
```

Comment:
Maybe add the fact that it's a private function and so cannot be called independently with any topic object?

Reply:
done